### PR TITLE
Clockwork cult rework: Post-merge micro-patch

### PR DESCRIPTION
**Pull request by ghost**
@KorPhaeron wanted the rework to just be playable ASAP and then merged once it was, so now that I don't have to worry about rushing that I can take a more precise hand in getting stuff done through slower testing.

This patch has some tiny stuff that was passed over to get it out in time or just plain missed:
* Ratvar won't indefinitely prolong the shuttle to give himself more time to decorate
* Recollection now (re)correctly shows `HONOR RATVAR` over and over with no categories when Ratvar has arrived
  * get it? recorrectly?
* War clock-golems can wear clothing, as was the original intention

I'd like this to be speedmerged since all the changes were intended to be packaged with the original rework, and also because of the current bug that prevents the round from ending, but I also imagine that the servers won't update by the 24h mark either way.

### CODE REVIEW

**Comment by ChangelingRain** [comment]
I mean, oranges merged it. Kor wanted more reviewing.

**Comment by AnonymousNow** [comment]
>War clock-golems
What's this when it's at home? Do we have brass golems now?

**Comment by Xhuis** [comment]
Brass golems are a thing outside of war clock-golems, yuh. You can only make them if you get your hands on some brass from a servant, but they exist and they're pretty good.

**Comment by KorPhaeron** [comment]
@tgstation/commit-access merging this now because rounds that don't end are bad

File: code/modules/mob/living/carbon/human/species_types/golems.dm [hunk 1]
```diff
@@ -632,6 +632,8 @@
 /datum/species/golem/clockwork/no_scrap //These golems are created through the herald's beacon and leave normal corpses on death.
```
Comment (KorPhaeron) [inline code/modules/mob/living/carbon/human/species_types/golems.dm]: These coulda just not been golems honestly

Comment (Xhuis) [reply code/modules/mob/living/carbon/human/species_types/golems.dm]: Mumble grumble consistency

Comment (KorPhaeron) [reply code/modules/mob/living/carbon/human/species_types/golems.dm]: They're gonna get messages from adamantium ones and stuff as well

**Pull request merged**